landing-pages/site/content/en/blog/airflow-2.10.0/index.md

```diff
@@ -0,0 +1,129 @@
+
+```python
+def hello_world():
+	print("hello world!")
+
+def hello_world_again():
+	print("hello world again!")
+
+with DAG(
+    dag_id="hello_worlds",
+    default_args={"executor": "LocalExecutor"},  # Applies to all tasks in the DAG
+) as dag:
+    # All tasks will use the executor from default args automatically
+    hw = hello_world()
+    hw_again = hello_world_again()
+```
+
+## Dynamic Dataset scheduling through DatasetAlias
+
+Airflow 2.10 comes with `DatasetAlias` class which can be passed as a value in the `outlets`, `inlets` on a task, and `schedule` on a DAG. An instance of `DatasetAlias` is resolved dynamically to a real dataset. Downstream can depend on either the resolved dataset or on an alias itself.
+
+`DatasetAlias` has one argument `name` that uniquely identifies the dataset. The task must first declare the alias as an outlet, and use `outlet_events` or `yield Metadata` to add events to it.
+
+### Emit a dataset event during task execution through outlet_events
+```python
+from airflow.datasets import DatasetAlias
+
+@task(outlets=[DatasetAlias("my-task-outputs")])
+def my_task_with_outlet_events(*, outlet_events):
+    outlet_events["my-task-outputs"].add(Dataset("s3://bucket/my-task"), extra={"k": "v"})
+```
+### Emit a dataset event during task execution by yielding Metadata
+```python
+from airflow.datasets.metadata import Metadata
+
+@task(outlets=[DatasetAlias("my-task-outputs")])
+def my_task_with_metadata():
+    s3_dataset = Dataset("s3://bucket/my-task}")
+    yield Metadata(s3_dataset, extra={"k": "v"}, alias="my-task-outputs")
+```
+
+There are two options for scheduling based on dataset aliases. Schedule based on `DatasetAlias` or real datasets.
+
+```python
+with DAG(dag_id="dataset-producer"):
+    @task(outlets=[Dataset("example-alias")])
+    def produce_dataset_events():
+        pass
+
+with DAG(dag_id="dataset-alias-producer"):
+    @task(outlets=[DatasetAlias("example-alias")])
+    def produce_dataset_events(*, outlet_events):
+        outlet_events["example-alias"].add(Dataset("s3://bucket/my-task"))
+
+with DAG(dag_id="dataset-consumer", schedule=Dataset("s3://bucket/my-task")):
+    ...
+
+with DAG(dag_id="dataset-alias-consumer", schedule=DatasetAlias("example-alias")):
+    ...
+```
+
+## Dark mode for Airflow UI
+
+Airflow 2.10 comes with new Dark Mode feature which is designed to enhance user experience by offering an alternative visual theme that is easier on the eyes, especially in low-light conditions. You can toggle the crescent icon on the right side of the navigation bar to switch between light and dark mode.
+
+![Airflow Dark mode](airflow_dark_mode.png)
+
+![Airflow Light mode](airflow_light_mode.png)
+
+## Contributors
+Thanks to everyone who contributed to this release, including Amogh Desai, Andrey Anshin, Brent Bovenzi, Daniel Standish, Ephraim Anierobi, Hussein Awala, Jarek Potiuk, Jed Cunningham, Jens Scheffler, Tzu-ping Chung, Vincent Beck, Wei Lee, and over 120 others!
```

Comment:
This is just a placeholder from the last post. I need to figure out how to get this.